config/create-gke-manifest.js

```diff
@@ -962,7 +962,7 @@ async function applyEditor() {
                 {
                   name: 'PEER_BY_DEFAULT',
                   value: 'true'
-                }
+                },
```

Comment:
You should add the environment variable here.
```
{
   name: 'IMG_MIN_SIZE_TO_RESIZE',
   value: '10'
}
```

Reply:
Also maybe rename the variable to `IMG_MIN_SIZE_TO_COMPRESS` for clarity

Reply:
the e2e test failed, when I added it. And passed, when I removed it.

Reply:
done.